source/MissionAction.cpp

```diff
@@ -122,6 +123,8 @@ void MissionAction::Load(const DataNode &node, const string &missionName)
 			else
 				child.PrintTrace("Error: Unsupported use of \"system\" LocationFilter:");
 		}
+		else if(key == "teleport")
```

Comment:
I think we also need to make sure hasValue is true (otherwise just writing teleport alone will trigger an error)

Reply:
Oh, right

Reply:
done